torchmd/run.py

```diff
@@ -286,7 +286,7 @@ def dynamics(args, mol, system, forces):
     for k in range(args.replicas):
         npy_name = os.path.join(args.log_dir, args.output + f"_{k}.npy")
         xyz_name = os.path.join(args.log_dir, args.output + f"_{k}.xyz")
-        converter_xyz_output(npy_name, xyz_name, mol.z)
+        xyz_writer(npy_name, xyz_name, mol.element)
```

Comment:
```py
npy_traj = np.load(npy_name)
molc = mol.copy()
molc.coords = npy_traj.astype(np.float32)
molc.write(xyz_file)
```

No need for a custom writer then I assume

Reply:
it's not going to work if you have a npzMolecule object; there is no copy and write function